core/views.py

```diff
@@ -85,3 +87,18 @@ def form_invalid(self, form):
             messages.error(self.request, error.as_data()[0].message)
 
         return self.get()
+
+
+class FeedbackView(LoginRequiredMixin, FormView):
+    template_name = 'feedback.html'
+    form_class = FeedbackForm
+
+    def form_valid(self, form):
+        message = mail.EmailMessage(form.cleaned_data['subject'],
```

Comment:
To cause less confusion here, you may wanna change `message` to `email` or `email_message`.